app/controllers/V1/assigned_counsel_claims_controller.rb

```diff
@@ -0,0 +1,22 @@
+module V1
+  class AssignedCounselClaimsController < ApplicationController
+    def update
+      assigned_counsel_claim.update!(allowed_params)
+      render json: assigned_counsel_claim, status: :created
+    rescue ActiveRecord::RecordInvalid, ActionController::ParameterMissing => e
+      render json: { errors: e.message }, status: :unprocessable_entity
```

Comment:
this isn't explicitly in the ticket but could we use the report_error method here so that this gets logged and reported in Sentry